app/views/documents/_filter_form.html.erb

```diff
@@ -1,21 +1,36 @@
 <% unless defined?(filters); filters = []; end %>
 <% document_type_plural = document_type.to_s.pluralize %>
+
 <div class="filter-form-block">
-  <div class="inner-block">
-    <h2><%= t('document_filters.description').html_safe %></h2>
-    <form method="get" action="<%= send(:"#{document_type_plural}_path") %>" id="document-filter" class="js-document-filter filter-form">
-      <fieldset>
-        <legend class="visuallyhidden"><%= t("support.filter") %> <%= document_type_plural %></legend>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: t('document_filters.description').html_safe,
+    font_size: "m",
+    heading_level: 2,
+    margin_bottom: 8
+  } %>
+
+  <form method="get" action="<%= send(:"#{document_type_plural}_path") %>" id="document-filter" class="js-document-filter filter-form">
+    <fieldset>
```

Comment:
You might be able to use this https://components.publishing.service.gov.uk/component-guide/fieldset

Reply:
Unfortunately, I don't think I can use it here because the legend is visible to screen readers only and I can't seem to find an option to do that with the component (sounds like it could be a useful enhancement for the future though).

Reply:
>the legend is visible to screen readers only
 
![image](https://user-images.githubusercontent.com/71266765/135093088-5ed3d34a-b4f9-4a4a-b365-7138cf8596aa.png)

Sorry, how do you mean? That the gem version legend needs to be different?